calliope/test/common/test_model/storage_cap.yaml

```diff
@@ -0,0 +1,106 @@
+
```

Comment:
Call this something like `group_constraints_storage_cap.yml` so its clearer what it tests?